### PR TITLE
[ng] Testing page: add required args to run angular_test

### LINKED ISSUE #792: Testing guide should use --test-arg=--platform
_From @StepFischer on June 30, 2017 11:22_

Hi,

I've tried the [advanced angular testing](https://webdev.dartlang.org/angular/guide/testing) with the [quickstart example](https://github.com/angular-examples/quickstart). Unfortunately executing `pub run angular_test` outputs:

```
The pub serve output is at /tmp/angular_test_pub_serve_output.log.
Run with --verbose to get this output in the console instead.
pub serve test --port=0
Pub "serve" started on http://localhost:46447
Finished compilation. Running tests...
pub run test --pub-serve=46447
No tests ran.
Shutting down...
`pub serve` was not terminated
```

Executing with `--verbose`` results in:

```
pub serve test --port=0
Loading source assets...
Loading angular2, angular2/transform/reflection_remover, test/pub_serve and dart_to_js_script_rewriter transformers...
Pub "serve" started on http://localhost:34653
Serving angular_app test on http://localhost:34653
Finished compilation. Running tests...
pub run test --pub-serve=34653
No tests ran.
Shutting down...
`pub serve` was not terminated
```

any clues, what's wrong?

_Copied from original issue: dart-lang/angular#480_

### PR DESCRIPTION
**Pull request by chalin**
The only essential change in this PR is the change in arguments used to invoke `angular_test`.

Fixes #792.
Contributes to #768.

Staged at https://angulardart-org-dev.firebaseapp.com/angular/guide/testing

The angular_test counterpart is https://github.com/dart-lang/angular/pull/493.

### CODE REVIEW

File: src/angular/guide/testing.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+the app component's `name` property to "World", expecting the infamous
+greeting as a result.
+
+The "Greet world HTML" test uses the fixture's `rootElement` to get the native
+DOM element generated from the component's template. It uses the DOM element
+to check that the app component generates the expected HTML.
+
+## Test the Hero Editor
+
+In this section you'll write tests for the [tutorial](../tutorial)'s [Hero Editor](../tutorial/toh-pt1).
```
Comment (kwalrath) [inline src/angular/guide/testing.md]: Overly long line here...

File: src/angular/guide/testing.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+To run tests, open a terminal, and from the project root issue the following
+command:
+
+<?code-excerpt class="code-shell"?>
+```sh
+  pub run angular_test --test-arg=--tags=aot --test-arg=--platform=dartium
+```
+
+The test framework runs code transformers, launches [pub serve][], loads the test file, and runs tests:
```
Comment (kwalrath) [inline src/angular/guide/testing.md]: Long line here...

File: src/angular/guide/testing.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+nextpage:
+  title: Glossary
+  url: /angular/glossary
+---
+Testing is an essential part of software development.
+This page shows how to use the **[angular_test][]** package to perform
+**[component/subsystem testing][testing]**.
+
+To track the state of all documentation on testing Dart web apps,
+subscribe to notifications on [issue #195](https://github.com/dart-lang/site-webdev/issues/195).
```
Comment (kwalrath) [inline src/angular/guide/testing.md]: I'd like to keep line length to <= 80 characters. Put the whole link on its own line?

**Comment by chalin** [comment]
Since this page will be replaced soon, I'll hold off on the line wrapping (unless we make some more edits before then).

**Pull request merged**